charts/gpu-device-plugin/templates/daemonset.yaml

```diff
@@ -0,0 +1,79 @@
+{{- if .Values.daemonSet.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-gpu-plugin
+  labels:
+    app: intel-gpu-plugin
+spec:
+  selector:
+    matchLabels:
+      app: intel-gpu-plugin
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app: intel-gpu-plugin
+    spec:
+      containers:
+      - name: intel-gpu-plugin
+        env:
```

Comment:
Please also add support to configure GPU plugin with its different modes: sharedDevNum, enableMonitoring, allocationPolicy and logLevel. No need for the "resourceManager" as it's being EoL'd.